<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game of Life Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Game of Life Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="gameTypeCombo">
          <item>
           <property name="text">
            <string>Simple</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Grid Size (HxW)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="heightSpin">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="widthSpin">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="fillCheck">
          <property name="text">
           <string>Fill</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createGameButton">
          <property name="text">
           <string>Create</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="GameWidget" name="gameWidget" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuColors">
     <property name="title">
      <string>Colors</string>
     </property>
     <addaction name="colorBlackWhiteAction"/>
     <addaction name="colorBlueOrangeAction"/>
     <addaction name="colorGreenRedAction"/>
    </widget>
    <widget class="QMenu" name="menuRAM">
     <property name="title">
      <string>RAM</string>
     </property>
     <addaction name="ramByteAction"/>
     <addaction name="ramKiloByteAction"/>
     <addaction name="ramMegaByteAction"/>
    </widget>
    <addaction name="menuColors"/>
    <addaction name="menuRAM"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="ramByteAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </action>
  <action name="ramKiloByteAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>kB</string>
   </property>
  </action>
  <action name="ramMegaByteAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MB</string>
   </property>
  </action>
  <action name="colorBlackWhiteAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Black / White</string>
   </property>
  </action>
  <action name="colorBlueOrangeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Blue / Orange</string>
   </property>
  </action>
  <action name="colorGreenRedAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Green / Red</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GameWidget</class>
   <extends>QWidget</extends>
   <header>gamewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
